tests/regression/lp-1865503/10-eth0.network

```diff
@@ -0,0 +1,11 @@
+[Match]
+Name=eth0
```

Comment:
Maybe this is the problem, could be not is actually handed down by the container manager.